components/Player/Player.tsx

```diff
@@ -48,8 +48,11 @@ const Player: React.FC<IPlayerProps> = ({
     [currentTrack.duration]
   );
   const isLastTrack = currentTrackIndex === tracks.length - 1;
-  const { url, previewUrl } = currentTrack;
+  const { url, previewUrl, transcripts, duration } = currentTrack;
```

Comment:
Question: do we GET/load transcripts even if the user/listener isn't using them?  What sort of bandwidth cost is that going to be?  I assume we're either paying to load these from a CDN, or more likely the Nextjs app sending the fairly large JSON to the browser.

Reply:
The only transcript data that gets loaded prior to the user opening the CC view is the the VTT, which is loaded by the browser when it encounters the track element. The VTT files have seemed to be fairly small, even for the longer Buzzcast transcript I have been working with. The choice to even use the track element was based on accessibility, since users may be using technologies that assist using the track data without having to use our CC feature.

Reply:
Larger transcript files, like the JSON format files, do not get loaded until the user opens the CC view.

Reply:
Cool, makes good sense.